packages/yup/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@availity/yup",
+  "version": "1.0.0",
+  "description": "Additional Methods for Yup validation library",
+  "main": "index.js",
+  "typings":"typings/index.d.ts",
+  "scripts": {
+    "test": "test"
+  },
+  "keywords": [
+    "tus",
+    "resumable",
+    "upload",
+    "availity"
+  ],
+  "author": "Kyle Gray <kyle.gray@availity.com>",
+  "license": "MIT",
+  "peerDependencies": {
+    "yup": "^0.27.0",
+    "moment": "^2.24.0"
```

Comment:
could dayjs be used since it is smaller?

Reply:
Actually it could be. I also considered [date-fns](https://www.npmjs.com/package/date-fns). I think `react-dates` is wanting to swap over to that as well.

This library is slightly biased in that it is assuming the user is wanting to validate with `react-dates` since `moment` is a required dep. I know there have been instances where people have alias'd `dayjs` and worked out.

Reply:
Swapped out those bad boys 🔋 📅